test/integration/gems_test.rb

```diff
@@ -71,9 +71,9 @@ class GemsSystemTest < SystemTest
 
   test "version navigation" do
     visit rubygem_version_path(@rubygem, "1.0.0")
-    click_link "Next version"
+    click_link "Next version →"
```

Comment:
I thought it's correct. This page links has a new text with arrows. Though tests passed without arrows. I'd better remove it? :)